src/ert/_c_wrappers/fm/ecl/ecl_run.py

```diff
@@ -246,9 +246,9 @@ def initMPI(self):
             machine_list = [localhost] * self.num_cpu
 
         self.machine_file = f"{self.base_name}.mpi"
-        with open(self.machine_file, "w") as fileH:
+        with open(self.machine_file, "w", encoding="utf-8") as filehandle:
             for host in machine_list:
-                fileH.write(f"{host}\n")
+                filehandle.write(f"{host}\n")
```

Comment:
In [this PR](https://github.com/equinor/ert/pull/3849/files) you called `f_handle`, should we use the same name?